features/mbedtls/targets/TARGET_CRYPTOCELL310/aes_alt.h

```diff
@@ -27,23 +27,13 @@
 extern "C" {
 #endif
 
-
-struct sw_aes_context
-{
-    int nr;                     /*!<  number of rounds  */
-    uint32_t *rk;               /*!<  AES round keys    */
-    uint32_t buf[68];           /*!<  unaligned data    */
-    int sw_key_set;
-};
-
 typedef struct
 {
-    struct {
-        SaSiAesUserContext_t CC_Context;
-        SaSiAesEncryptMode_t CC_cipherFlag;
-        SaSiAesUserKeyData_t CC_KeyData;
-    } cc_aes_context;
-    struct sw_aes_context sw_context;
+
+    int keysize_nr;             /*number of rounds for SW use case, and key size for CC use case*/
```

Comment:
Indeed, you can. This was only a minor suggestion because renaming this back to `nr` means that there will be a lot less differences between the mbed TLS code and the CC `aes_alt.c` code.